### PR TITLE
Fix the link for the SharePoint management shell

### PR DESCRIPTION
**Pull request by derhallim**
Management shell link is broken. Related to Issue #2648

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by derhallim** [comment]
Resolved merge conflicts.

File: SharePoint/SharePointOnline/sharepoint-app-bar.md [hunk 1]
```diff
@@ -205,8 +205,8 @@ You can temporarily disable the SharePoint app bar in your tenant to prepare for
 <br>
 Temporarily disable the SharePoint app bar:
 
+1.	Download the latest version of [SharePoint Online Management Shell](https://www.microsoft.com/en-us/download/details.aspx?id=35588).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-app-bar.md:208]:
```suggestion
1.	Download the latest version of [SharePoint Online Management Shell](https://www.microsoft.com/download/details.aspx?id=35588).
```

Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-app-bar.md:208]:
```suggestion
1.	Download the latest version of [SharePoint Online Management Shell](https://www.microsoft.com/download/details.aspx?id=35588).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the locale from the link.

**Comment by scanum** [comment]
@Holland-ODSP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**